src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const listItems = list.querySelectorAll('li');
+
+listItems.forEach(li => {
+  const span = document.createElement('span');
+
+  li.prepend(span);
+
+  span.append(span.nextSibling);
+});
+
+list.addEventListener('click', (e) => {
+  const item = e.target.closest('li').children[1];
```

Comment:
Add a check for 'e.target.closest('li')' to ensure it's not null before accessing its children property.